autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Looks like you need to add some newlines if you'd like to format this as a bulleted list.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Not sure what happened there.  Thanks.</p>